Tests/CouchDBTests/DBTests.swift

```diff
@@ -36,25 +36,28 @@ class DBTests: CouchDBTest {
     
     /// Test that the database can be created and deleted.
     func testDB() {
-        couchDBClient.createDB(dbName) {(db: Database?, error: NSError?) in
-            if let error = error {
-                XCTFail("DB creation error: \(error.code) \(error.localizedDescription)")
-            }
-
-            guard let db = db else {
-                XCTFail("Created database is nil")
-                return
-            }
-
-            print(">> Database successfully created")
-            
-            self.couchDBClient.deleteDB(db) {(error: NSError?) in
+        setUpDatabase() {
+            self.couchDBClient.deleteDB(self.dbName) {(error: NSError?) in
                 if let error = error {
                     XCTFail("DB deletion error: \(error.code) \(error.localizedDescription)")
                 }
-                print(">> Database successfully deleted")
+                print(">> Database successfully deleted ")
+            }
+            self.couchDBClient.createDB(self.dbName) {(db: Database?, error: NSError?) in
```

Comment:
Is this right? It looks like `createDB` and `deleteDB` will potentially be executed in parallel.  The delete used to be invoked during the completion for create, since those callbacks could (potentially) be invoked asynchronously.